app/locale/en.js

```diff
@@ -6671,16 +6671,17 @@ module.exports = {
     },
     hoc_page: {
       hoai_activities_poetry_title: 'AI HackStack: Poetry in Motion',
-      hoai_activities_poetry_level: 'Grades 6+ | Multimodal AI | ELA',
       hoai_activities_poetry_description: 'Bring poetry to life with AI HackStack! Learners explore a poem’s theme, craft prompts to generate images, and evaluate AI outputs for bias while blending ELA skills with AI literacy as words transform into art.',
       hoai_activities_social_title: 'AI HackStack: Social Media Showdown',
-      hoai_activities_social_level: 'Grades 6+ | Multimodal AI | Algebra',
+      hoai_labels_1: 'Grades 6+',
```

Comment:
I'm not a fan of this 1,2,3 in key names. They should be descriptive enough. You could have probably created a key like ela, algebra instead